.github/workflows/realease-on-pr-merge.yaml

```
@@ -0,0 +1,54 @@
+          if [ "$BUMP_TYPE" = "major" [; then
+            npm version major
+          elif [ "$BUMP_TYPE" = "minor" ]; then
+            npm version minor
+          elif [ "$BUMP_TYPE" = "patch" ]; then
+            npm version patch
+          fi
+          git push origin HEAD
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: ${{ steps.bump_version.outputs.new_tag }}
```

actionlint

property "bump_version" is not defined in object type {get_labels: {conclusion: string; outcome: string; outputs: {string => string}}; version_type: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)